cypress/plugins/index.js

```diff
@@ -12,7 +12,7 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-import webpack from '@cypress/webpack-preprocessor'
+const webpack = require('@cypress/webpack-preprocessor')
```

Comment:
This line was changed recently, can somebody explain why? We can't use import in this file for now without adding some webpack loader configuration.

Reply:
Well the commit  https://github.com/neherlab/covid19_scenarios/commit/1ee53fcdda989967b9a6c9175953efafac5d25d1 was by @abrie  with the comment ' Change 'require' to 'import' to fix eslint no-var-requires.'

Reply:
Thanks @noleti.
I checked that PR but I don't know the reason, was this line breaking the build or something? Maybe @abrie can explain which part I need to change (worst case I'll just add a //eslint ignore).

Reply:
I assume it was just eslint complaining: https://palantir.github.io/tslint/rules/no-var-requires/

Reply:
@Joluma indeed it _was_ breaking the build because of a hard limit to eslint errors. But the fix breaks the 2e2 tests, which I don't think anybody has been running (until now). I agree with your solution to explicitly ignore the error.